runtime/src/main/java/org/corfudb/runtime/view/replication/IReplicationProtocol.java

```diff
@@ -1,6 +1,9 @@
 package org.corfudb.runtime.view.replication;
 
+import com.google.common.collect.Streams;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.Streams'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)